api/src/main/kotlin/edu/wgu/osmt/keyword/KeywordController.kt

```diff
@@ -138,4 +146,130 @@ class KeywordController @Autowired constructor(
         return found
     }
 
+    @PostMapping(
+        path = ["${RoutePaths.API}${RoutePaths.API_V3}${RoutePaths.KEYWORD_SKILLS}"],
+        produces = [MediaType.APPLICATION_JSON_VALUE]
+    )
+    @ResponseBody
+    @PreAuthorize("isAuthenticated()")
+    fun searchCategorySkills (
```

Comment:
Why is this endpoint referring in the Path that is related to Keywords and its name is specific for Categories only?

Reply:
Good catch, I extracted the code from category controller